[2023-03-18T15:14:28.176+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: data_ingestion_gcs_dag.local_to_gcs manual__2023-03-18T15:13:52.344423+00:00 [queued]>
[2023-03-18T15:14:28.184+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: data_ingestion_gcs_dag.local_to_gcs manual__2023-03-18T15:13:52.344423+00:00 [queued]>
[2023-03-18T15:14:28.184+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2023-03-18T15:14:28.185+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 1
[2023-03-18T15:14:28.185+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2023-03-18T15:14:28.196+0000] {taskinstance.py:1300} INFO - Executing <Task(PythonOperator): local_to_gcs> on 2023-03-18 15:13:52.344423+00:00
[2023-03-18T15:14:28.202+0000] {standard_task_runner.py:55} INFO - Started process 1010 to run task
[2023-03-18T15:14:28.205+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'data_ingestion_gcs_dag', 'local_to_gcs', 'manual__2023-03-18T15:13:52.344423+00:00', '--job-id', '1874', '--raw', '--subdir', 'DAGS_FOLDER/ingestion_script_gcp.py', '--cfg-path', '/tmp/tmpzxmnvrfk']
[2023-03-18T15:14:28.206+0000] {standard_task_runner.py:83} INFO - Job 1874: Subtask local_to_gcs
[2023-03-18T15:14:28.215+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/settings.py:249: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  SQL_ALCHEMY_CONN = conf.get("database", "SQL_ALCHEMY_CONN")

[2023-03-18T15:14:28.251+0000] {task_command.py:388} INFO - Running <TaskInstance: data_ingestion_gcs_dag.local_to_gcs manual__2023-03-18T15:13:52.344423+00:00 [running]> on host 4b3eeaf199d3
[2023-03-18T15:14:28.308+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=data_ingestion_gcs_dag
AIRFLOW_CTX_TASK_ID=local_to_gcs
AIRFLOW_CTX_EXECUTION_DATE=2023-03-18T15:13:52.344423+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-03-18T15:13:52.344423+00:00
[2023-03-18T15:14:28.319+0000] {ingestion_script_gcp.py:47} INFO - Loading GCS bucket, <google.cloud.storage.client.Client object at 0x7f05bc94e290>
[2023-03-18T15:14:55.447+0000] {python.py:177} INFO - Done. Returned value was: None
[2023-03-18T15:14:55.464+0000] {taskinstance.py:1323} INFO - Marking task as SUCCESS. dag_id=data_ingestion_gcs_dag, task_id=local_to_gcs, execution_date=20230318T151352, start_date=20230318T151428, end_date=20230318T151455
[2023-03-18T15:14:55.512+0000] {local_task_job.py:208} INFO - Task exited with return code 0
[2023-03-18T15:14:55.539+0000] {taskinstance.py:2578} INFO - 0 downstream tasks scheduled from follow-on schedule check
